int32 bias_x
int32 bias_y
string label